routing/turns.cpp

```diff
@@ -290,6 +300,12 @@ bool IsLaneWayConformedTurnDirectionApproximately(LaneWay l, CarDirection t)
   }
 }
 
+bool IsLaneUnrestricted(const SingleLaneInfo & lane)
+{
+  /// @todo Is there any reason to store None single lane?
```

Comment:
Won't it break something?

Reply:
Should not, keep it for future thinking.